CONTRIBUTING.md

```diff
@@ -24,6 +24,26 @@ requests.
 Major projects are tracked [here](https://github.com/rust-bitcoin/rust-lightning/projects).
 Major milestones are tracked [here](https://github.com/rust-bitcoin/rust-lightning/milestones?direction=asc&sort=title&state=open).
 
+Getting Started
+---------------
+
+First and foremost, start small.
+
+This doesn't mean don't be ambitious with the breadth and depth of your contributions but rather
+understand the project context and culture before investing an asymmetric number of hours on
+development compared to your merged work.
+
+Even if you have an extensive open source background or sound software engineering skills, consider
+that code comprehension by the other set of regular contributors is as much important as its
```

Comment:
Would recommend rephrasing "the other set of regular contributors" simply as "the reviewers". Tightening it up makes the connection between "comprehension" and "technical correctness" more apparent without loss of meaning.

Reply:
Thanks, updated at 1dfb2f9

Reply:
Sorry about nit picking this to death. After reading the new version, I think the following wording would be clearer:

> that the reviewers' comprehension of the code is as much important as technical correctness.

Reply:
That's a minute update, that's fine :)